packages/react-i18n/src/utilities/money.ts

```diff
@@ -4,9 +4,10 @@ export function getCurrencySymbol(
   locale: string,
   options: Intl.NumberFormatOptions,
 ) {
-  const delimiters = ',.';
+  const delimiters = ',.٫';
+  const numerals = '0٠';
   const directionControlCharacters = /[\u200E\u200F]/;
-  const numReg = new RegExp(`0[${delimiters}]*0*`);
+  const numReg = new RegExp(`[${numerals}][${delimiters}]*[${numerals}]*`);
```

Comment:
Thanks for this, I didn't know about these!
I will update my PR description and changelog to reflect that I'm now using these, and I also ran into a use case with the negative sign and direction control characters that I needed to account for. At the same time, this resulted in preserving the negative sign returned by [Intl.NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat), rather than substituting with our own, which has broader implications on `formatCurrency` output.